openedx/tests/completion_integration/test_services.py

```diff
@@ -1,18 +1,20 @@
 """
 Tests of completion xblock runtime services
 """
+from completion.models import BlockCompletion
+from completion.services import CompletionService
```

Comment:
Please help me add the new test for `vertical_is_complete` function into this file.